lib/multiple_search_selection.dart

```diff
@@ -739,8 +758,8 @@ class _MultipleSearchSelectionState<T>
         if (widget.title != null) ...[
           widget.title!,
         ],
-        if (pickedItems.isNotEmpty)
-          widget.pickedItemsContainerBuilder?.call([
+        if (widget.pickedItemsContainerBuilder != null)
```

Comment:
i wanted to use my widget even when the picked items list was empty

Reply:
Ok this makes sense.